Makefile

```diff
@@ -124,6 +124,8 @@ install:
 	cp ./assets/wall_4K.png ${PREFIX}/share/hyprland
 	cp ./assets/wall_8K.png ${PREFIX}/share/hyprland
 
+	rst2man ./docs/hyprctl.1.rst | gzip -c > /usr/share/man/man1/hyprctl.1.gz
```

Comment:
what the fuck is rst2man and what is this a dependency of

Reply:
I think it's python-docutils on arch.

Docutils-common on debian

Reply:
It just coverts rst to a man page format. Groff is a terrible format to read.

Reply:
I'd rather want us not to rely on python, if it's possible.

Reply:
We can use `pandoc --from rst --to man <files>`

Reply:
I'm okay with using pandoc. I've used it before. Not too many differences. Just some slight format changes.

Reply:
I've updated this to use pandoc instead of rst2man.

A notable good thing from the switch is that we can use switches in the Makefile for setting the version and date.

Reply:
Oh, even better! Good work.

Reply:
Thank you. I'm going to have to switch over to pandoc on my own projects.

This should be good to go.
Eventually I'll write up a man page for the config too, but that's going to take a bit.

Reply:
pandoc is pandoc. haskell but already compiled.